wordpress-mobile/WordPress-iOS
# Send all translations to gutenberg-mobile

Fixes part of: https://github.com/wordpress-mobile/gutenberg-mobile/issues/203

2 new props are sent to gutenberg editor during initialization:
1. Current language's short code(like en, es etc)
2. Gutenberg localized strings key value map from InfoPlist.strings file for the current language

Gutenberg related translations will be added to InfoPListTranslator.m which then will be parsed into InfoPlist.strings files, that's why I am only parsing InfoPlist.strings in this PR and not from any other localization strings files.

**To test**
Check out the branch from https://github.com/wordpress-mobile/gutenberg-mobile/pull/536
cd to the gutenberg-mobile path on terminal and run yarn install, yarn start
Run `rake dependencies` on WPiOS folder
Run XCode and open the WPiOS app on simulator
Put a breakpoint to initial props on JS side
Open a gutenberg post
See that the file contents from InfoPlist.strings on iOS side is passed to there.

(since we don't have the real gutenberg translations added to InfoPlist.strings I could just add some developer style tests, but feel free to edit/extend the tests)

Update release notes:
